src/pybind/mgr/orchestrator_cli/module.py

```diff
@@ -20,6 +20,12 @@ class OrchestratorCli(MgrModule):
             "desc": "List devices on a node",
             "perm": "r"
         },
+        {
+            'cmd': "orchestrator service ls "
+                   "name=svc_type,type=CephString,req=false",
```

Comment:
Could make this a CephChoices and enumerate the services (unless we want to maintain flexibility?)

Reply:
I think we want to avoid having the client vet the type string,  Given that the mgr does all of the real processing here, keeping the client as "dumb" as possible means less dependencies between them. That said, I'm not religious about this, so if you feel strongly about it I can change it.